src/components/NFTEditionSelect.vue

```diff
@@ -1,128 +1,58 @@
-        <NFTEditionSelectItem
-          v-for="(item, index) in items"
-          :key="index"
-          :name="item.name"
-          :currency="item.currency"
-          :price-label="item.priceLabel"
-          :stock="item.stock"
-          :is-selected="item.value === selectedValue"
-          :current-price="item.price"
-          :default-price="item.defaultPrice"
-          @click="handleClickPriceSelectItem(item)"
-        />
-      </tbody>
-    </table>
-
+  <div class="flex items-center gap-[12px] w-full flex-wrap">
     <div
-      :class="[
-        'flex',
-        'flex-wrap',
-        'justify-end',
-        'items-stretch sm:items-center',
-        'gap-[12px]',
-        'flex-col sm:flex-row',
-      ]"
-    >
-      <ButtonV2
-        v-if="!isAllSoldOut"
-        preset="plain"
-        class="text-white underline max-638:order-1"
-        :text="
-          $t(
-            isSingleItem
-              ? 'nft_edition_view_edition_button_text'
-              : 'nft_edition_select_compare_button_text'
-          )
-        "
-        @click="handleClickCompareItemsButton"
-      />
-      <template v-if="isSingleItem || isAllSoldOut">
-        <span v-if="isSingleItem && !items[0].name" class="text-white">{{
-          priceLabel
-        }}</span>
-        <NFTStockLabel
-          v-if="items.length > 1 || isAllSoldOut"
-          :stock="stock"
-          :is-dark="!isAllSoldOut"
-        />
-      </template>
-
+      class="hidden laptop:block border-b-[1px] border-[#ebebebeb] w-full my-[16px]"
+    />
+    <div class="text-[18px] font-500 text-dark-gray w-full">
+      {{ $t('nft_edition_select_section_label') }}
+    </div>
+    <NFTEditionSelectItemV2
+      v-for="(item, index) in items"
+      :key="index"
+      :name="item.name"
+      :currency="item.currency"
+      :price-label="item.priceLabel"
+      :stock="item.stock"
+      :is-selected="item.value === selectedValue"
+      :current-price="item.price"
+      :default-price="item.defaultPrice"
+      @click="handleClickPriceSelectItem(item)"
+    />
+    <div class="flex items-center gap-[12px] w-full mt-[12px]">
       <ButtonV2
         v-if="
-          !isAllSoldOut &&
-            selectedItem.price > 0 &&
-            !selectedItem.isPhysicalOnly
+          !isAllSoldOut && selectedItem.price > 0 && !selectedItem.hasShipping
         "
         :is-disabled="!selectedItem"
-        preset="secondary"
-        :text="$t('nft_edition_select_confirm_button_text_gift')"
-        @click="handleClickGiftButton"
+        preset="primary"
+        class="w-[58%] laptop:w-[260px]"
+        :text="$t('nft_edition_select_confirm_button_text_add_to_cart')"
+        @click="handleClickAddToCartButton"
       >
-        <template #prepend>
-          <IconGift class="w-[16px]" />
-        </template>
+        <IconAdd class="w-[16px]" />
+        <p class="ml-[8px]">
+          {{ $t('nft_edition_select_confirm_button_text_add_to_cart') }}
+        </p>
```

Comment:
use `v-text` for content

Reply:
[💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)